org.musescore.MuseScore.json

```diff
@@ -0,0 +1,89 @@
+{
+  "app-id": "org.musescore.MuseScore",
+  "runtime": "org.kde.Platform",
+  "runtime-version": "5.9",
+  "sdk": "org.kde.Sdk",
+  "command": "musescore",
+  "rename-desktop-file": "mscore.desktop",
+  "rename-appdata-file": "mscore.appdata.xml",
+  "rename-icon": "mscore",
+  "finish-args": [
+    "--device=dri",
+    "--share=network",
+    "--share=ipc",
+    "--socket=x11",
+    "--socket=pulseaudio",
+    "--filesystem=home",
+    "--env=QT_QPA_PLATFORM=flatpak",
+    "--env=QT_QPA_FLATPAK_PLATFORM=xcb"
+  ],
+  "build-options": {
+    "cflags": "-O2 -g",
+    "cxxflags": "-O2 -g",
+    "env": {
+      "V": "1"
+    }
+  },
+  "modules": [
+    {
+      "name": "qtwebkit",
+      "buildsystem": "simple",
+      "cleanup-platform": [
+        "/bin",
+        "/mkspecs"
+      ],
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://github.com/qt/qtwebkit/archive/v5.212.0-alpha2.tar.gz",
+          "sha256": "6db43b931f64857cfda7bcf89914e2730b82164871a8c24c1881620e6bfdeca1"
+        }
+      ],
+      "build-commands": [
+        "qmake",
+        "grep -rl '/usr/local' . | xargs sed -i 's|/usr/local|/app|g'",
+        "make",
+        "find . -name \"cmake_install.cmake\" | xargs sed -i 's|/usr|/app|g'",
+        "make install"
+      ],
+      "cleanup": [
+        "/include",
+        "/lib/cmake",
+        "/lib/pkgconfig"
+      ]
+    },
+    "shared-modules/lame/lame-3.99.5.json",
+    {
+      "name": "musescore",
+      "buildsystem": "cmake",
+      "config-opts": [
+        "-DCMAKE_BUILD_TYPE=RELEASE",
+        "-DBUILD_ALSA=OFF",
+        "-DBUILD_JACK=OFF",
+        "-DBUILD_PORTAUDIO=OFF"
+      ],
+      "build-commands": [
+        "make lrelease",
+        "make",
+        "make install PREFIX=/app"
```

Comment:
These shouldn't be needed?

Reply:
`make lrelease` needs to be run manually, or `make install` fails at translations.

Reply:
You can use the `make-args` property probably.